File: command-line-arguments/java/CommandLineArgs.java
public class CommandLineArgs {
    public static void main(String[] args) {
        if (args.length == 0) 
            System.out.println("No command line arguments were specified");
        else {
            for (String arg : args) {
                System.out.println(arg);
            }
        }
    }    
}

File: command-line-arguments/python/CommandLineArgs.py
import sys

if (len(sys.argv) == 0):
    print('No command line arguments were specified')
else:
    # Note:
    # sys.argv[0] is the first argument represents the script name, hence starting the loop with index 1
    for arg in range(1, len(sys.argv)):
        print(arg)